pynicotine/pluginsystem.py

```diff
@@ -261,7 +261,7 @@ def load_enabled(self):
             return
 
         to_enable = self.config.sections["plugins"]["enabled"]
-        log.add_debug("Loading plugin(s): %s" % ', '.join(to_enable))
```

Comment:
I understand what you're trying to do, but I interpreted this as loaded & enabled for a split second.

Reply:
_"Loaded" appears after each individual plugin init, so "Loading" status is not relevant in the debug log.
This is outcome of my changes:_
```
00:48:36 Loading plugin system
00:48:36 [Misc] Enabled plugin(s): leech_detector, multipaste, port_checker, testreplier, youtube_noembed
00:48:36 [Misc] Stored setting 'num_downloads' is no longer present in the 'Leech Detector' plugin
00:48:36 Leech Detector: Complain to leechers after tolerating 20 downloads, require users have minimum 10 files in 1 shared public folders.
00:48:36 Loaded plugin Leech Detector
00:48:36 Loaded plugin Multi Paste
00:48:36 Loaded plugin Port Checker
00:48:36 Loaded plugin Test Replier
00:48:36 Loaded plugin YouTube Noembed
```